recipes/pymedphys_analysis/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymedphys_analysis" %}
+{% set version = "0.8.1" %}
+{% set file_ext = "tar.gz" %}
```

Comment:
Only name and version should be set per parameters.

Reply:
Alright.